samples/cpp/continuous_batching_accuracy/continuous_batching_accuracy.cpp

```diff
@@ -78,7 +78,9 @@ int main(int argc, char* argv[]) try {
     // vLLM specific params
     scheduler_config.max_num_seqs = 2;
 
-    ov::genai::ContinuousBatchingPipeline pipe(models_path, scheduler_config);
+    // It's possible to construct a Tokenizer from a different path.
+    // If the Tokenizer isn't specified, it's loaded from the same folder.
+    ov::genai::ContinuousBatchingPipeline pipe(models_path, ov::genai::Tokenizer{models_path}, scheduler_config);
```

Comment:
To use default arg for tokenizer?

Reply:
I don't get the question.

I wanted to test that constructor. And I thought it can be kept for educational purposes.